config_parser_test.cc

```diff
@@ -76,3 +69,12 @@ TEST_F(NginxStringConfigTest, BlankConfig) {
 TEST_F(NginxStringConfigTest, CommentsOnlyConfig) {
   EXPECT_FALSE(ParseString("# This comment does nothing"));
```

Comment:
I suggest adding comments on failure: ex. << "Error: failed to detect comment"; on these tests for more clarity when they are fired.